modules/secrets_manager/secrets.tf

```diff
@@ -0,0 +1,22 @@
+locals {
+  gitlab_secrets_arns = var.integration_type == "gitlab" ? [
+    aws_secretsmanager_secret.spectral_dsn[0].arn,
+    aws_secretsmanager_secret.gitlab_token[0].arn,
+    aws_secretsmanager_secret.gitlab_webhook_secret[0].arn
+  ] : []
+}
+
+resource "aws_secretsmanager_secret" "gitlab_webhook_secret" {
```

Comment:
we are supporting only gitlab?

Reply:
Currently yes, because only the source code of GitLab supports secrets from secrets manager, in the future more bots would supports as well